aikau/src/main/resources/alfresco/renderers/Thumbnail.js

```diff
@@ -188,6 +263,40 @@ define(["dojo/_base/declare",
       lastThumbnailModificationProperty: "jsNode.properties.cm:lastThumbnailModification",
 
       /**
+       * This indicates whether or not the aspect ratio of the thumbnail will be retained. This means that
+       * for non-square images there will be white-space between the image and its frame. If this is configured
+       * to be false then portrait thumbnails will be clipped at the bottom and landscape thumbnails will be 
+       * vertically stretched. However, if no [dimensions]{@link module:alfresco/renderers/Thumbnail#dimensions}
+       * are provided then the aspect ration will always be maintained.
+       *
+       * @instance
+       * @type {boolean}
+       * @default
+       * @since 1.0.40
+       */
+      maintainAspectRatio: true,
```

Comment:
Got some problems with the use of "aspect ratio" in general. Changing the aspect ratio of an image certainly could be applied to cropping it, but it more normally refers to stretching it. I'd prefer to see terms such as fill/crop and fit/letterbox, which more clearly describe what's happening with the image, which is not actually changing, only the window through which it's being viewed. Realise this is subjective, but don't want to say nothing when it makes me feel so uncomfortable as-is.